cmd/perennial_branches.go

```diff
@@ -0,0 +1,47 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Originate/git-town/lib/git"
+	"github.com/spf13/cobra"
+)
+
+var branchToAdd string
+var branchToRemove string
+
+var perennialBranchesCommand = &cobra.Command{
+	Use:   "perennial-branches",
+	Short: "Displays or updates your perennial branches",
+	Run: func(cmd *cobra.Command, args []string) {
+		if branchToAdd != "" {
+			git.EnsureHasBranch(branchToAdd)
+			git.EnsureIsNotMainBranch(branchToAdd, fmt.Sprintf("'%s' is already set as the main branch", branchToAdd))
+			git.EnsureIsNotPerennialBranch(branchToAdd, fmt.Sprintf("'%s' is already a perennial branch", branchToAdd))
+			git.AddToPerennialBranches(branchToAdd)
+			return
+		}
+
+		if branchToRemove != "" {
+			git.EnsureIsPerennialBranch(branchToRemove, fmt.Sprintf("'%s' is not a perennial branch", branchToRemove))
```

Comment:
Extract this block into a `removePerennialBranch(branchName)` method in this file.